staging/src/k8s.io/api/batch/v1beta1/types.go

```diff
@@ -26,20 +26,6 @@ import (
 // +k8s:prerelease-lifecycle-gen:introduced=1.8
 // +k8s:prerelease-lifecycle-gen:deprecated=1.22
```

Comment:
drop these comments, they are messing up code generation for JobTemplateSpec

Reply:
Alright. I got everything working and the CI looks good.